src/Plugins/ComplexCore/src/ComplexCore/Filters/Algorithms/PartitionGeometry.cpp

```diff
@@ -244,6 +246,7 @@ Result<> PartitionGeometry::partitionCellBasedGeometry(const IGridGeometry& inpu
 
   ParallelData3DAlgorithm dataAlg;
   dataAlg.setRange(dims[0], dims[1], dims[2]);
+  dataAlg.setParallelizationEnabled(false);
```

Comment:
Check if parallelization disabled is intended.

Reply:
@mmarineBlueQuartz Does this need to be disabled? Can we do the same checks for out-of-core arrays versus in-core arrays?